tests/sentry/tasks/test_weekly_reports.py

```diff
@@ -559,37 +555,38 @@ def test_message_builder_multiple_users_prevent_resend(self, message_builder, re
         user2 = self.create_user()
         self.create_member(teams=[self.team], user=user2, organization=self.organization)
 
-        event1 = self.store_event(
-            data={
-                "event_id": "a" * 32,
-                "message": "message",
-                "timestamp": iso_format(self.three_days_ago),
-                "stacktrace": copy.deepcopy(DEFAULT_EVENT_DATA["stacktrace"]),
-                "fingerprint": ["group-1"],
-            },
-            project_id=self.project.id,
-        )
+        with self.options({"issues.group_attributes.send_kafka": True}):
```

Comment:
What does this option do?

Reply:
It enables some kafka processing so that the table join actually works - without it the tests would fail cause it wouldn't do the join and the query would be incorrect. I believe this is a temporary thing until we can release it fully to self hosted.